### PR TITLE
[12.0][MIG][WIP] stock_picking_report_valued

### PR DESCRIPTION
**Pull request by antoniorubiopesol**

### CODE REVIEW

**Review by angelmoya — APPROVED** [review]
Tested on Runbot and code review

LGTM :+1:

File: stock_picking_report_valued/models/stock_move_line.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+from odoo import api, fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = 'stock.move.line'
+
+    sale_line = fields.Many2one(
+        related='move_id.sale_line_id', readonly=True,
+        string='Related order line',
+        related_sudo=True,  # See explanation for sudo in compute method
```
Comment (rousseldenis) [inline stock_picking_report_valued/models/stock_move_line.py]: related_sudo is True by default. Please remove all entries

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Minor comment

File: stock_picking_report_valued/models/stock_move_line.py (hunk 1 above)
Comment (angelmoya) [reply stock_picking_report_valued/models/stock_move_line.py]: @rousseldenis I did it

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review.

Don't hesitate to complete the migration using https://github.com/OCA/maintainer-tools#readme-generator.

You have to change manifest 'website' key at least. https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

File: stock_picking_report_valued/readme/BUGTRACKER.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/stock-logistics-reporting/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
```
Comment (Change2improve) [inline stock_picking_report_valued/readme/BUGTRACKER.rst:4]: I think this file is not necessary, see https://github.com/OCA/maintainer-tools/tree/master/template/module

File: stock_picking_report_valued/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===========================
+Stock Picking Report Valued
+===========================
+
```
Comment (Change2improve) [inline stock_picking_report_valued/readme/DESCRIPTION.rst:8]: I think you should remove the first lines and just let the description. See https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/DESCRIPTION.rst

File: stock_picking_report_valued/readme/MAINTAINER.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
```
Comment (Change2improve) [inline stock_picking_report_valued/readme/MAINTAINER.rst:12]: I think this file is not necessary, see https://github.com/OCA/maintainer-tools/tree/master/template/module

File: stock_picking_report_valued/readme/USAGE.rst [hunk 5]
```diff
@@ -0,0 +1,12 @@
+To get the stock picking valued report:
+
+#. Create a Sale Order with stockable products a *Valued picking* able
+   customer.
+#. Confirm the Sale Order.
+#. Click on *Deliveries* stat button.
+#. Go to *Print > Delivery Slip*.
+
```
Comment (Change2improve) [inline stock_picking_report_valued/readme/USAGE.rst:8]: I think you should remove the last lines and just let the usage description. See https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/USAGE.rst

**Review by Change2improve — CHANGES_REQUESTED** [review]
Minor changes in /readme

**Comment by novawish** [comment]
Just asking: Can we add the computations of total amount on `stock.move` as well? (using the same method as `stock.picking`)

PS: Also, can we make all the fields stored?

**Comment by rousseldenis** [comment]
@antoniorubiopesol Do you plan to attend comments ?

**Comment by emagdalenaC2i** [comment]
@antoniorubiopesol You should take a look at #60 and #59 

And if you can't move forward with this migration, please tell it so someone else can finish it.

**Comment by pedrobaeza** [comment]
Superseeded by #63

**Pull request closed without merging**